ciongke/Makefile

```diff
@@ -98,4 +104,15 @@ sinker-image:
 sinker-deployment:
 	kubectl apply -f sinker_deployment.yaml
 
-.PHONY: hook-image hook-deployment hook-service test-pr-image sinker-image sinker-deployment
+deck-image:
```

Comment:
We can but I'm always wary of intelligent Makefiles. I'd rather have some duplication than confusion :P